src/cli.ts

```diff
@@ -317,7 +317,7 @@ async function logBalanceInfo(ctx: Context) {
   log(`P-chain ${ctx.pAddressBech32}: ${pbalance} FLR`)
 }
 
-function logNetworkInfo(ctx: Context) {
+export function logNetworkInfo(ctx: Context) {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums